src/port1.0/portmain.tcl

```diff
@@ -113,6 +114,7 @@ default destroot {${destpath}}
 default filesdir files
 default revision 0
 default epoch 0
+default reproducible_timestamps 1
```

Comment:
Although using `1` probably works, MacPorts convention is to use `yes` and `no` for Booleans.
```suggestion
default reproducible_timestamps yes
```